templates/default/customer/customeradd.html

```diff
@@ -21,67 +21,6 @@ <H1>{$layout.pagetitle}</H1>
 //-->
 </SCRIPT>
 <style>
-
-    .customer-header {
-        width: 100%;
-        display: flex;
-        flex-direction: row;
-        justify-content: flex-start;
-        align-items: center;
-        flex-wrap: wrap;
-    }
-
-    .customer-header-left-panel {
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        align-items: flex-start;
-        flex-wrap: wrap;
-        padding-left: 0.5em;
-        padding-right: 0.5em;
-        flex-basis: 420px;
-    }
-
-    .customer-header-right-panel {
-        display: flex;
-        padding-left: 0.5em;
-        padding-right: 0.5em;
-    }
-
-    .customer-header label{
-        padding-right: 0.5em;
-    }
-
-    .customer-header label > *:not(:first-child){
-        padding-right: 0.5em;
-    }
-    .customer-header-row {
-        display: flex;
-        min-width: 100%;
-        padding-bottom: 0.5em;
-    }
-
-    .customer-header-row input{
-        flex-grow: 1;
-    }
-
-    .customer-data {
-		width: 100%;
-		display: flex;
-		flex-direction: row;
-		justify-content: flex-start;
-		align-items: flex-start;
-		flex-wrap: wrap;
-	}
-
-	.customer-data > div {
-		flex-grow: 1;
-	}
-
-	#customer-right-panel {
-		margin-right: 2em;
-	}
-
     @media screen and (max-width: 470px) {
```

Comment:
To usunięcie sporego kodu CSS na pewno nie wprowadza żadnej regresji?

Reply:
No właśnie na tym polegała cała zmiana. #customer-data zastąpione zostało .lmsbox-panels, natomiast #customer-right-panel oraz #customer-left-panel zastąpione zostało przez .lmsbox-panel